.travis.yml

```diff
@@ -70,7 +70,7 @@ jobs:
       - yarn chromatic --auto-accept-changes --app-code="dlpro96xybh" # v1
       - yarn chromatic --auto-accept-changes # v2
       - yarn build-storybook
-      - npm config set //registry.npmjs.org/:_authToken=$NPM_PUBLISH_TOKEN && yarn lerna publish --yes --canary --preid next --pre-dist-tag next
+      - npm config set //registry.npmjs.org/:_authToken=$NPM_PUBLISH_TOKEN && yarn lerna publish --yes --canary --preid next --dist-tag next
```

Comment:
`--dist-tag` applies to all releases and `--canary` releases are always prereleases, so the `pre-` was unnecessary here.

Reply:
Don't we need to `yarn build`?

Reply:
Not sure if it _really_ matters, but I would put this under a deploy provider like we do in the `tag` stage. It's technically a deployment of code right?

Reply:
Ha - that would help. Done